### PR TITLE
[Cloud Tasks] Add HTTP task sample

### PR DESCRIPTION
**Pull request by averikitsch**
Test projects will have to be granted permission for HTTP targets or wait until whitelisting is off.

### CODE REVIEW

File: tasks/phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright 2018 Google LLC.
```
Comment (bshaffer) [inline tasks/phpunit.xml.dist]: nit: 2019! And should be for all new files!

File: tasks/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+5.  Create a Queue
+    To create a queue using the Cloud SDK, use the following gcloud command:
+    ```sh
+    gcloud beta tasks queues create-app-engine-queue my-appengine-queue
+    ```
+6. Set environment variables:
+
+    First, your project ID:
+
+        export PROJECT_ID=my-project-id
```
Comment (bshaffer) [inline tasks/README.md]: Setting environment  variables for PROJECT_ID, LOCATION_ID, and QUEUE_ID are not required, as we pass them into the script directly. The only environment variable required *should be* `GOOGLE_APPLICATION_CREDENTIALS`

**Comment by averikitsch** [comment]
Thanks for the review. Can I go ahead and merge?

**Comment by bshaffer** [comment]
Looks great!! Thank you!

**Pull request merged**